team.html

```diff
@@ -1,342 +1,356 @@
+            </div>
+        </section>
+        <section class="container">
+            <!-- TODO: make this into a script + template + json -->
+            <div class="row">
+                <h2>Advisors</h2>
+                <p class="inner">The advisors of UWNRG assist with many of the projects at UWNRG. The advisors comprise of professors at the University of Waterloo who provide resources for research and experimentation for UWNRG.</p>
+                 <div class="content-middle">
+                    <div class="profile">
+                        <div class="col-md-3">
+                            <img src="images/team/Mustafa_Yavuz.jpg"/>
+                        </div>
+                        <div class="col-md-9">
+                            <h4>Dr. Mustafa Yavuz</h4>
+                            <p>
+                                Professor Mustafa Yavuz envisions a production tool that can harness new types of magnetic templates and exploit them to make radically new devices and materials. These templates are an enabling tool for self-assembly manufacture/production of magnetic particles into new nano-structured macro-materials, with various applications the field of nanotechnology.
+                                <br/><small>More about him can be found <a href="https://uwaterloo.ca/institute-nanotechnology/people-profiles/mustafa-yavuz">here</a></small>
+                            </p>
+                        </div>
+                    </div>
+                    <div class="profile">
+                        <div class="col-md-3">
+                            <img src="images/team/Bo_Cui.jpg"/>
+                        </div>
+                        <div class="col-md-9">
+                            <h4>Dr. Bo Cui</h4>
+                            <p>
+                                Cui leads the Waterloo Nanofabrication Group whose research is focused on nano- and microstructure fabrication using Nano-Imprint Lithography (NIL) and electron beam lithography, thin film deposition and etching, with applications in biomedical, nanoelectric and other areas.
+                                <br/><small>More about him can be found <a href="https://uwaterloo.ca/institute-nanotechnology/people-profiles/bo-cui">here</a></small>
+                            </p>
+                        </div>
+                    </div>
+                 </div>
+            </div>
+            <div class="row">
+                <h2>Directors</h2>
+                <h3>Managing Director</h3>
+                <!-- <p class="inner">The Executive team consists of the project and technical director who oversee and organize all of the group's activities.</p> -->
+                 <div class="content">
+                    <div class="profile">
+                        <div class="col-md-3">
+                            <img src="images/team/Qianshu_Wang.jpg"/>

```

Comment:
So I noticed that only the names and pictures changed but somehow the entire file changed. I can't tell what happened, could you redo this to make sure the only changes are the names and pictures?